force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +519,13 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+
+        if (person1.getAge() > person2.getAge()) {
+            return person1;
+        } else if (person2.getAge() > person1.getAge()) {
+            return person2;
+        } else {
+            return person1; // Replace null with the variable you used to store the result
```

Comment:
You can simplify this a lot. If there is a tie, you can return either one. Can you solve this with one if statement and a return statement?